com.github.linthesia.linthesia.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "com.github.linthesia.linthesia",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "linthesia",
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gnome.GConf",
+        "--persist=.local/linthesia"
+    ],
+    "modules": [
+        "shared-modules/SDL/SDL-1.2.15.json",
+        "shared-modules/intltool/intltool-0.51.json",
+        "shared-modules/dbus-glib/dbus-glib.json",
+        {
+            "name": "gconf",
+            "config-opts": ["--disable-orbit"],
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/GConf/3.2/GConf-3.2.6.tar.xz",
+                    "sha256": "1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c"
+                }
+            ]
+        },
+        {
+            "name": "linthesia",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/linthesia/linthesia/archive/refs/tags/0.7.0.tar.gz",
+                    "sha256": "9c46b8b4a6abffa283a85e2096d6ab5c845404d2b471d8c91db36f01c7b5861c"
+                }
+            ]
+        },
+        {
+            "name": "metainfo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m0644 com.github.linthesia.linthesia.metainfo.xml /app/share/metainfo/com.github.linthesia.linthesia.metainfo.xml",
+                "install -D -m0644 linthesia.png /app/share/app-info/icons/flatpak/128x128/com.github.linthesia.linthesia.png"
```

Comment:
definitely **not** there.

Icons are installed in `${FLATPAK_DEST}/share/icons/hicolor/128x128/apps`